app/assets/javascripts/controllers/header/header_controller.js

```diff
@@ -20,13 +21,24 @@ function HeaderCtrl($scope, eventNotifications, $timeout) {
   var refresh = function() {
     $timeout(function() {
       vm.newNotifications = eventNotifications.state().unreadNotifications;
+      updateTooltip();
     });
   };
 
   var destroy = function() {
     eventNotifications.unregisterObserverCallback(refresh);
   };
 
+  function updateTooltip() {
+    var unreadNotificationCount = 0;
+    if (angular.isArray(vm.notificationGroups)) {
+      angular.forEach(vm.notificationGroups, function(group) {
+        unreadNotificationCount += group.unreadCount;
+      });
+    }
+    vm.notificationsIndicatorTooltip = __(unreadNotificationCount + " unread notifications");
```

Comment:
This doesn't work i18n-wise .. `__(unreadNotificationCount + " unread notifications")` needs to be either `unreadNotificationCount + __(" unread notifications")` or, possibly better, something like `sprintf(__("%d unread notifications"), unreadNotificationCount)`